include/simd.h

```diff
@@ -433,10 +503,104 @@ namespace vargas {
   }
   __RG_STRONG_INLINE__
   int16x16 blend(const int16x16 &mask, const int16x16 &t, const int16x16 &f) {
```

Comment:
Aside from new functionality/extensions, I also found that _mm256_blendv_epi8 was being called instead of epi16 for the uint16_t types, which meant that a wrong function was being called previously when working with 16-bit integers for blend.